### DISCUSSION TITLE
SUGGESTION: "Confidence" embedded in wav file name

### DISCUSSION THREAD

**Discussion by CaiusX** [mcguirepr89/BirdNET-system#28] [Ideas]
Hi All

Do you think there is a case for having a value for confidence score embedded in the wav filename, 
e.g

211-Laughing_Dove_YYYY-MM-DD-birdnet-**89%**-time.wav

Would be useful for analytics (what species are identified with highest average confidence etc.).

Best

**Comment by mcguirepr89** [comment]
I can do that -- it will be on the back burner for now, but those values are readily available and should be easily incorporated. It wouldn't have the percentage marker, since that would throw a real curve ball into these scripts, but the value itself can be in there somewhere, probably dash-separated. (I'll add it to the Improvements project)
